home/static/css/mystyles.css

```diff
@@ -137,18 +106,18 @@ div.bar-container {
   font-size: 1.5em;
   padding-bottom: 5px;
 }
-.post-comments {
-  display: block;
-}
 .close-frame {
   position: absolute;
   right: 10px;
 }
 .close-frame a:link {
   text-decoration: none;
-  color:gray; 
+  color:gray;
 }
 .close-frame a:visited {
   text-decoration: none;
   color:gray;
-}
\ No newline at end of file
+}
+.feed-icon {
```

Comment:
Looks like the CSS didn't get deployed for some reason, and I'm [unable to re-deploy](https://devcenter.heroku.com/articles/cedar-14-migration) stuff onto staging. The CSS changes do appear on localhost.